test/quantum_mechanics/schrodinger/time_dependent_equation_in_one_dimension_test.py

```diff
@@ -0,0 +1,40 @@
+from collections import namedtuple
+from pytest import fixture
+from sympy import exp, I, pi
+from symplyphysics.core.expr_comparisons import expr_equals
+from symplyphysics.laws.quantum_mechanics.schrodinger import (
+    time_dependent_equation_in_one_dimension as wave_function_eqn,
+)
+
+Args = namedtuple("Args", "m u psi x t")
+
+
+@fixture(name="test_args")
+def test_args_fixture() -> Args:
+    position = wave_function_eqn.position
+    time = wave_function_eqn.time
+    m = 1
+    u = 0
+    psi = -3 * exp(I * (position - time / 2))
+    x = 0.1
+    t = 2
```

Comment:
Try using some real values and non-zero potential, because your test somehow missed a bug in your formula.